setup.py

```diff
@@ -41,7 +41,7 @@
 
```

Comment:
above, should add `Programming Language :: Python :: 3.5`, and remove the 3.4 unless we test with it.
also should add a `python_requires` key (see the [packaging guide](https://packaging.python.org/guides/distributing-packages-using-setuptools/#python-requires) for reference)